dev/com.ibm.ws.threading/src/com/ibm/ws/threading/internal/ThreadPoolController.java

```diff
@@ -716,6 +764,19 @@ enum LastAction {
         }
     }
 
+    /**
+     * Switch to regular pool sizing after startup completes
+     */
+    void startupCompleted() {
+        initialStartupCompleted = true;
+        setPoolSize(coreThreads);
+        // no need to run the controller cycle if there will be no pool size changes
+        if (coreThreads < maxThreads) {
+            activeTask = new IntervalTask(this);
+            timer.schedule(activeTask, interval, interval);
+        }
```

Comment:
The task may already be running from the constructor.  Why starting it a second time?

Reply:
Good catch. I've added a check for (activeTask == null) before starting it up. 
Also added an else clause to stop the controller cycle if it was running during startup (because startupPoolSize < maxThreads) but should not run after startup if (coreThreads == maxThreads).